### ISSUE TITLE
The `value` attribute of the `<data>` element is required

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#811]
## Describe the bug

[Spec says](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-data-element),

>The `value` attribute must be present. Its value must be a representation of the element's contents in a machine-readable format.

Markuplint currently does not warn if the `value` attribute is omitted.
https://playground.markuplint.dev/#JTNDIWRvY3R5cGUlMjBodG1sJTNFJTBBJTNDaHRtbCUzRSUwQSUzQ2hlYWQlM0UlMEElMDklM0N0aXRsZSUzRURvY3VtZW50JTNDJTJGdGl0bGUlM0UlMEElM0MlMkZoZWFkJTNFJTBBJTNDYm9keSUzRSUwQSUwOSUzQ2gxJTNFJTYwZGF0YSU2MCUyMGVsZW1lbnQlMjB0ZXN0JTNDJTJGaDElM0UlMEElMEElMDklM0NkYXRhJTNFdGV4dCUzQyUyRmRhdGElM0UlMEElM0MlMkZib2R5JTNFJTBBJTNDJTJGaHRtbCUzRSUwQQ==

Nu Html Checker showed an error.

## Code Example

```html
👎 <data>text</data>
👍 <data value="text">text</data>
```

**Comment by YusukeHirao** [comment]
@SaekiTominaga 

My bad. It was simply my mistake. I overlooked the implementation.
Thank you for reporting it.
